packages/dara-components/js/common/button/button.tsx

```diff
@@ -69,7 +70,8 @@ const StyledButton = injectCss(styled(UiButton)<StyledButtonProps>`
  */
 function Button(props: ButtonProps): JSX.Element {
     const [style, css] = useComponentStyles(props);
-    const [onClick, loading] = useAction(props.onclick);
+    const onClick = useAction(props.onclick);
+    const loading = useActionIsLoading(props.onclick.loading);
```

Comment:
Technically `props.onclick` can be `None` I believe, this should have a null check 
```suggestion
    const loading = useActionIsLoading(props.onclick?.loading);
```

Reply:
It shouldn't even have `.loading` here 🤦‍♂️. The null check is in the hook...